apiv2/routers/leads.py

```
@@ -0,0 +1,1129 @@
+    message: str
+
+router = APIRouter(tags=["Leads"])
+
+def force_strings(choices):
+    return [(key, str(value)) for key, value in choices]
+
+
+@router.get("/meta/", response_model=Dict[str, Any])
+def get_leads(
+    request: Request,
+    token_payload: dict = Depends(verify_token)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)